teeracle/src/lib.rs

```diff
@@ -97,6 +145,15 @@ pub mod pallet {
 		}
 	}
 }
+impl<T: Config> Pallet<T> {
+	fn is_whitelisted(mrenclave: [u8; 32]) -> bool {
+		Self::whitelist().iter().any(|m| *m == mrenclave)
+	}
+
+	pub fn whitelisted_oracle_count() -> u32 {
```

Comment:
why a helper function for this? the name is longer than the actual direct instruction